### ISSUE TITLE
[Bug] dimension error raised in VariationalStrategy

### ISSUE THREAD

**Issue by sjiang2018** [cornellius-gp/gpytorch#1090]
# 🐛 Bug

my inducing point has the dimension as  inducing point size: torch.Size([10, 100, 6])
where 10 is the number of inducing point
6 is the input channel
and 100 is the number input sequences.
so the model would be 
output(1*2) =  model(intput(100*6))
## To reproduce
however, in variational_strategy.py
line 86: full_inputs = torch.cat([inducing_points, x], dim=-2)
line 91: num_induc = inducing_points.size(-2)
In my case, the dimension -2 is not the number of inducing point.
** Code snippet to reproduce **
```python
# Your code goes here
# Please make sure it does not require any external dependencies (other than PyTorch!)
# (We much prefer small snippets rather than links to existing libraries!)
```

** Stack trace/error message **
```
// Paste the bad output here!
```

## System information

**Please complete the following information:**
- 
- 
-
